seed/php-sdk/seed.yml

```diff
@@ -9,10 +9,12 @@ test:
     buildCommand:
       - pnpm --filter @fern-api/php-sdk dist:cli
     runCommand: node sdk/dist/bundle.cjs
-
 language: php
 generatorType: SDK
 defaultOutputMode: github
 fixtures: {}
-scripts: []
+scripts:
+  - docker: composer:2.7.9
+    commands:
+      - composer install && composer build && composer analyze && composer test
```

Comment:
Note that we're running _all_ of the scripts defined in the generated `composer.json` file, so we'll know whenever anything is wrong with the generated code. 

We do this in a single line to reduce the number of containers spun up.